pkg/did/doc.go

```diff
@@ -130,7 +130,7 @@ func populateRawPublicKey(pk *PublicKey) (map[string]interface{}, error) {
 			if err != nil {
 				return nil, err
 			}
-		case ECKeyType:
+		case P256KeyType:
```

Comment:
Ay specific reason to narrow this down ? The idea was it could support all the EC curves - P256, P384, P521. Since the code is generic for all EC curves. Now when we add new curve(eg P384) in edge-service, we would need to update this project as well to support it.